core/src/main/kotlin/com/exactpro/th2/read/db/bootstrap/Main.kt

```diff
@@ -45,6 +45,7 @@ import com.google.common.util.concurrent.ThreadFactoryBuilder
 import com.google.protobuf.UnsafeByteOperations
 import com.opencsv.CSVWriterBuilder
 import io.netty.buffer.Unpooled
+import io.netty.util.internal.StringUtil.toHexString
```

Comment:
I believe we should not use Netty's internal stuff. Could you please use `io.netty.buffer.ByteBufUtil.hexDump(byte[])` instead if it matches the requirements?